aip/aog/3001.md

```diff
@@ -0,0 +1,127 @@
+---
+aip:
+  id: 3001
+  state: draft
+  created: 2019-10-03
+  updated: 2019-10-03
+  type: process
+js:
+  - /assets/js/graphviz/viz.js
+  - /assets/js/graphviz/lite.render.js
+  - /assets/js/aip/aip-graphviz.js
+permalink: /3001
+---
+
+# Actions on Google AIP Process
+
+This AIP extends [AIP-1][] with details specific to Actions on Google AIPs
+("AoG AIPs"). Any details of [AIP-1][] not modified or contradicted by this AIP
+also apply to AoG AIPs.
```

Comment:
Same "branding" comment as in the other PR, we should always spell out (but aog is fine in urls or id to abbreviate)